cypress/integration/statusSpec.js

```diff
@@ -1,14 +1,7 @@
-import { testResponseCode, testContentType } from '../support/metaTestHelper';
-
-describe('Simorgh Status', () => {
-  it('should return 200', () => {
-    testResponseCode('/status', 200);
```

Comment:
We're getting rid of this because we're killing off the Simorgh endpoints, right?

I worry about this slightly due to referencing this URL in the run book.

Reply:
Since we changed the test base path to `https://www.bbc.com`, the `/status` endpoint does not reflect the Simorgh application's status any longer.